Master

marcusengel opened the pull request.

jeduan committed:
Updates frameworks

jeduan committed:
Merge pull request #24 from thomashilzendegen/master

Init of FB SDK in browser proxy

jeduan committed:
1.2.2

jeduan committed:
Adding Facebook iOS version 4.5.1 #19

jeduan committed:
1.3.0

jeduan committed:
Changes for iOS9

jeduan committed:
1.4.0-0

jeduan committed:
Logs app activation. Fixes #17

jeduan committed:
feed method is now an alias for share

jeduan committed:
Updating Bolts to v1.2.2. Fixes #33

jeduan committed:
1.4.0-1

jeduan committed:
Updating to Facebook SDK v4.6

jeduan committed:
1.4.0

Commit:
https://github.com/jeduan/cordova-plugin-facebook4/issues/42

jeduan committed:
Merge pull request #45 from gillesperrin/master

https://github.com/jeduan/cordova-plugin-facebook4/issues/42

Commit:
added better callbacks for appInvite on iOS

Changed appInvite callbacks, you can now determine if the user pressed
done, cancel or really finished the invite. iOS only, no android know
how!

Commit:
fixed callback error

Commit:
remove whitespace

jeduan committed:
Merge pull request #60 from nightstomp/master

added better callbacks for appInvite on iOS

jeduan committed:
Revert "added better callbacks for appInvite on iOS"

jeduan committed:
Merge pull request #61 from jeduan/revert-60-master

Revert "added better callbacks for appInvite on iOS"

Commit:
Merge remote-tracking branch 'jeduan/master'

Commit:
better appInvite callbacks

iOS only: determine when user canceled, pressed done button or really
invited someone

Commit:
Stop crashes when handleError is called

When handleError is called without a context (this can happen when graphApi has invalid permissions), the plugin crashes.

jeduan committed:
Merge pull request #62 from nightstomp/master

better appInvite Callbacks (iOS only)

jeduan committed:
Merge pull request #67 from kriserickson/master

Stop crashes when handleError is called

jeduan committed:
iOS SDK v4.7

jeduan committed:
SPDX licence compatibility

jeduan committed:
Now automatically updates the XCode deployment target to the minimum required by Facebook

jeduan committed:
Updates NSAppTransportSecurity

jeduan committed:
Moves AppInviteDialogDelegate implementation

jeduan committed:
Remove deprecated API for showing dialogs

 - Uses typedef blocks to improve legibility

jeduan committed:
Small refactor on responseObject

jeduan committed:
loginManager is now a singleton

yanca232 committed:
Set plugin's onload param to true to enable install attribution

jeduan committed:
Merge pull request #90 from RazorGator/master

Set plugin's onload param to true to enable install attribution

dtritus committed:
Fixed issues with AppInvite callbacks on Android

jeduan committed:
Revert "Now automatically updates the XCode deployment target to the minimum required by Facebook"

This reverts commit d75e2407a7efcd0186d5d79291d53693d26a3ca3.

As commented in https://github.com/jeduan/cordova-plugin-facebook4/commit/d75e2407a7efcd0186d5d79291d53693d26a3ca3
this wont work automagically

Commit:
fix problem with cached tokens

 which will prevent login. should fix #91 and #92

Commit:
completionHandler is optinal,just call it with nil

jeduan committed:
Merge pull request #98 from Hirbod/patch-1

fix problem with cached tokens

NGumby committed:
Fixes issue that prevents presenting the facebook window

The facebook login would not show up when used with an InAppBrowser.  This fixes the problem.

Fixes Warning: Attempt to present * on *whose view is not in the window hierarchy.

NGumby committed:
Merge pull request #1 from compusport/Fix-when-used-with-InAppBrowser

Fixes issue that prevents presenting the facebook window

yoav-zibin committed:
Update README.md

Removed warning that "this plugin uses Gradle and PhoneGap Build does not support that yet"

jeduan committed:
Merge pull request #101 from yoav-zibin/master

Update README.md

dtritus committed:
Pass AppInvite dialog result to success callback

jeduan committed:
Merge pull request #94 from dtritus/master

Fixed issues with AppInvite callbacks on Android

jeduan committed:
1.5.0

pragunvohra committed:
Added ios and android method to manually activate app with facebook events

This is an enhancement we’ve added to our fork of the repo. Changes
copied from
https://github.com/Anu2g/cordova-plugin-facebook4/commit/cde6356bfedcd15
4ef1c29facaeeab273cc4ba7b.

dorulazar committed:
Updated share_open_graph to use Open Graph objects along with properties and Open Graph actions with properties.

jeduan committed:
Merge pull request #122 from DoruLa/master

Updated share_open_graph.

jeduan committed:
Linting

jeduan committed:
Adding README for manual activation events

jeduan committed:
Merge branch 'pr/121'

jeduan committed:
1.6.0

jeduan committed:
1.6.1

mkorganashvili committed:
Send App Request Does Not Return Results

The json object is missed as a parameter in success function

mkorganashvili committed:
Set up activity result callback before call show

Callback is called only once, when you call the show() for the first time.
It needs to be set up before call the function.

jeduan committed:
Fixes #139

jeduan committed:
v1.6.2

jeduan committed:
Merge pull request #130 from mkorganashvili/patch-1

Send App Request Does Not Return Results

jeduan committed:
Merge pull request #134 from mkorganashvili/patch-2

Set up activity result callback before call show

jeduan committed:
1.6.3

jcvalerio committed:
Update ConnectPlugin.java

Success/error callbacks are not getting called on Android
- Set up activity result callback before call show

NGumby committed:
Merge remote-tracking branch 'refs/remotes/originmaster/master'

jeduan committed:
Merge pull request #142 from jcvalerio/master

Update ConnectPlugin.java

taina0407 committed:
Update ios plugin implement FBSDKGameRequestDialogDelegate

update gameRequest dialog callback error on user canceled

Commit:
added support for getDeferredApplink

Commit:
Fixed issue that grantedPermissions inspection always does not exist

pbakondy committed:
Set minSdkVersion preference

jeduan committed:
1.6.4

jeduan committed:
Merge pull request #159 from T2123893/master

Fixed issue that grantedPermissions inspection always does not exist

jeduan committed:
1.6.5

jeduan committed:
Merge pull request #156 from taina0407/master

update ios showDialog method apprequests callback

jeduan committed:
1.7.0

jeduan committed:
Merge pull request #99 from compusport/master

Fix issue when using with InAppBrowser on ios9

Commit:
Set Facebook Android SDK version to 4.8.+

jeduan committed:
Merge pull request #171 from TheCopartnery/master

Set Facebook Android SDK version to 4.8

divyekhanna committed:
Update package.json

divyekhanna committed:
Update plugin.xml

Commit:
Fix app_users filter

Commit:
Merge pull request #161 from pbakondy/minSdkVersion

Set minSdkVersion preference

Commit:
Merge pull request #185 from platogo/fix-app-users-filter

Fix app_users filter

Commit:
Merge pull request #174 from divyekhanna/master

Bump version number after PR #171

Commit:
Update README.md

Commit:
fix #102

grmmph committed:
Add share_native and share_feedWeb as share parameters

grmmph committed:
Fix indetation

grmmph committed:
Fix indetation

Commit:
added support for getDeferredApplink

Commit:
sdk upgrade to v4.11.0

nihsuy committed:
Fix feed empty content bug on android

grmmph committed:
Update README

Commit:
Merge pull request #207 from grmmph/master

Add share_native and share_feedWeb as share parameters

Commit:
#223 fix for signing in as different user

Sign the previous user out before trying to sign-in

pragunvohra committed:
Add OpenURL handler for ios9

pragunvohra committed:
I think the annotation is just 0x0

robertarnesson committed:
Update README.md

Commit:
iOS: Added support for share Open Graph dialog

robertarnesson committed:
Merge pull request #211 from nihsuy/fix/android-feed-empty-content

Fix feed empty content bug on android

GabrielCTroia committed:
Upgrade SDKCore

GabrielCTroia committed:
Upgrade SDKLoginKit

GabrielCTroia committed:
Upgrade FBSDKShareKit & FBSDKMessengerShareKit

robertarnesson committed:
Merge pull request #252 from jackpocket/ios_sdk_v4.11.13

Upgrade the iOS SDK to 4.13.1

robertarnesson committed:
Update plugin.xml

4.13.1

robertarnesson committed:
don't include facebook-native.js for browser platform

robertarnesson committed:
Merge pull request #241 from edskeizer/master

iOS: Added support for share Open Graph dialog

jkassis committed:
Merge remote-tracking branch 'upstream/master'

jkassis committed:
send deferred links back to app for handling

tushar-bidchat committed:
Fixed Warnings Due to Deprecated APIS

jkassis committed:
android support for deferredAppLinks

robertarnesson committed:
Merge pull request #270 from tushar-bidchat/master

Fixed Warnings Due to Deprecated APIS

marfire committed:
Add clarifying note to docs about changing APP_ID

Commit:
Updating Android and iOS SDK.

Added "quote" property for the share dialogs.

Bumped version number to 1.7.2 to reflect the change.

dudeofawesome committed:
Fixing broken link in README.md

dudeofawesome committed:
Fixing broken link in browser/README.md

dudeofawesome committed:
Merge remote-tracking branch 'origin/patch-2' into patch-1

Commit:
Merge pull request #281 from marfire/master

Add clarifying note to docs about changing APP_ID

Commit:
Merge pull request #291 from dudeofawesome/patch-1

Fixing broken links in README.md

Commit:
Merge remote-tracking branch 'refs/remotes/jeduan/master'

Commit:
Added forgotten semicolon

Commit:
1.7.2

robertarnesson committed:
fix NullPointerException

robertarnesson committed:
Merge pull request #288 from TheBosZ/master

Expose the "quote" property on share dialogs

robertarnesson committed:
Merge pull request #224 from hoffination/master

#223 fix for signing in as different user

robertarnesson committed:
Merge pull request #226 from pragunvohra/newOpenURL

Add OpenURL handler for ios9 API

Commit:
simplified browser setup

robertarnesson committed:
file:// protocol fix

Commit:
buffer calls until js sdk is ready

Commit:
1.7.3

robertarnesson committed:
js sdk v2.7

Commit:
ios sdk 4.15.1

Commit:
ios sdk 4.15.1

Commit:
1.7.4

FME-nstokoe committed:
Merge branch 'master' of github.com:jeduan/cordova-plugin-facebook4

# Conflicts:
#	.gitignore

FME-nstokoe committed:
references to jeduan

robertarnesson committed:
Merge pull request #362 from nstokoe/jeduan-mergeable

fetchDeferredAppLink (jkassis pr fixes)

simokhalil committed:
Avoid FB prompt if required permission declined

antonfire committed:
Update facebook-native.js

antonfire committed:
Update FacebookConnectPlugin.h

antonfire committed:
Update FacebookConnectPlugin.m

antonfire committed:
Update README.md

antonfire committed:
Update README.md

simllll committed:
Update facebook-browser.js

simllll committed:
Update facebook-browser.js

simllll committed:
fix __fbSdkReady calls

robertarnesson committed:
Merge pull request #384 from simllll/master

Update facebook-browser.js

jQrgen committed:
Updated outdated link to repo/docs

oak-tree committed:
upgrade to the last facebook sdk + a fix 

Fix the `Attribute activity#com.facebook.FacebookActivity@theme` error 

    Its better to just remove android:theme="@android:style/Theme.Translucent.NoTitleBar" from the <activity> xml tag since the latest Facebook SDK 4.16.1 already includes it.
source:  http://stackoverflow.com/a/39786339/1211174

jkervine committed:
Fix link from contributing to troubleshooting

jkervine committed:
Add fix documentation

jkervine committed:
Proper linking

jeduan committed:
Merge pull request #434 from jkervine/document_ios10_login_issue

Document ios10 login issue

jeduan committed:
Merge pull request #398 from CNATURE/master

upgrade to the last facebook sdk + a fix

jeduan committed:
Merge pull request #392 from jQrgen/patch-1

README: Updated outdated link to repo/docs

jeduan committed:
Merge pull request #375 from simokhalil/master

Avoid FB prompt if required permission declined

jeduan committed:
Merge pull request #382 from antonfire/master

Added function in iOS to check if a passed array of permissions have been granted

jeduan committed:
1.8.0

mpk2 committed:
Added hashtag support (#418)

* Added hashtag support

* Typo fix

* Misplaced the command earlier -- rather than typo

* Added hashtag for share dialog options example

jacquesdev committed:
Update plugin version in plugin.xml to 1.8.0 (#485)

Alexious-sh committed:
ios sdk 4.22.0 (#481)

jeduan committed:
SDK 4.22.1

jeduan committed:
1.9.0

dnoliver commented:
hello everybody, what if we solve this conflicts and merge this PR? I can do the resolve step if everybody agrees

jacquesdev committed:
Update plugin version in plugin.xml to 1.9.0 (#509)

jeduan committed:
Update plugin.xml to 1.9.1

jeduan committed:
1.9.1

vojtakopal committed:
application:openURL needs to return BOOL value (#548)

Not returning boolean in application:openURL has an undefined behaviour.

Commit:
Updated logPurchase to handle decimal values correctly. (#500)

* Updated logPurchase to handle decimal values correctly.

* Updated logPurchase to handle decimal values correctly.

RobertYoung committed:
Updated the ConnectPlugin so that it accepts multiple async requests (#468)

darkovac committed:
Fix for facebook-browser.js (#561)

* Update facebook-browser.js

* Commens and improvement on FB.login functon

Added comments and improved logic to check for response first then or response.status.

peterpeterparker committed:
Update Facebook SDK 4.31.1 for iOS 11 (#634)

* Update FacebookSDKs-iOS-4.31.1

* contentTitle, imageURL and contentDescription have been deprecated in Facebook API 2.9

* Update README according deprecation

peterpeterparker committed:
Initial import

peterpeterparker committed:
Add notice for cordova-android >= 7 compatibility

peterpeterparker committed:
Spelling

peterpeterparker committed:
Update for next version v1.10.0

peterpeterparker committed:
Add example to install previous version

peterpeterparker committed:
Spelling

peterpeterparker committed:
Update iOS 11.3

peterpeterparker committed:
Release v1.10.0 - Update Facebook iOS SDK

peterpeterparker committed:
Change sentences order

peterpeterparker committed:
Fix versioning

skaman committed:
Add ANDROID_SDK_VERSION preference (#646)

Add an installation variable in order to overwrite, if necessary and in case on conflict, the Facebook SDK version for Android

peterpeterparker committed:
Add note regarding PR #646

peterpeterparker committed:
Format

peterpeterparker committed:
Open changelog v1.11.0

peterpeterparker committed:
Format

peterpeterparker committed:
Format

peterpeterparker committed:
Format

peterpeterparker committed:
#645: Remove App Invites

peterpeterparker committed:
Next version 1.11.0

peterpeterparker committed:
App Invites removal is a breaking change, next version 2.0.0

peterpeterparker committed:
Add notice for older Cordova CLI

peterpeterparker committed:
#568: Fix null pointer exception crashing app after 2nd login

peterpeterparker committed:
#653: Update Facebook iOS SDK v4.33.0

peterpeterparker committed:
Release v2.1.0

jeduan committed:
Create issue_template.md (#668)

@jeduan awesome thx

peterpeterparker committed:
Typings and a bit of more details

peterpeterparker committed:
Typings

peterpeterparker committed:
Update Facebook iOS SDK 4.34.0

peterpeterparker committed:
Release v2.2.0

peterpeterparker committed:
Update Facebook iOS SDK 4.35.0

peterpeterparker committed:
Update to v2.3.0

peterpeterparker committed:
Release v2.3.0

peterpeterparker committed:
Update Facebook iOS SDK 4.36.0

peterpeterparker committed:
Release v2.4.0

peterpeterparker committed:
refactor(#706): Rename ANDROID_SDK_VERSION to FACEBOOK_ANDROID_SDK_VERSION

peterpeterparker committed:
doc(#708): Add a note regarding the Graph API version

peterpeterparker committed:
feat(#708): Sync Facebook SDK for iOS and Android

peterpeterparker committed:
lib(#713): update Facebook SDK for iOS

peterpeterparker committed:
lib(#713): use Facebook SDK v4.37.0 for Android too

peterpeterparker committed:
release: v3.1.0

peterpeterparker committed:
lib(#725): update Facebook SDK v4.38.1 for iOS

peterpeterparker committed:
lib(#725): update Facebook SDK v4.38.1 for Android

peterpeterparker committed:
release: v3.2.0

peterpeterparker committed:
merge: release v3.2.0 and Facebook SDK v4.38.1

adipascu committed:
Support for cordova-android v7 without workaround (#716)

peterpeterparker committed:
release: v3.3.0

peterpeterparker committed:
release: v4.0.0 (replace v3.3.0 as it might be a breaking change for some users)

msencer committed:
Enable app events to be sent from the pixel for both iOS and Android (#678) (#732)

peterpeterparker committed:
feat(#678): add FB key to log msg

peterpeterparker committed:
release: v4.1.0

peterpeterparker committed:
lib: update Facebook SDK v4.40.0

peterpeterparker committed:
merge(#743): Facebook SDK v4.40.0

peterpeterparker committed:
release: v4.2.0

regevbr committed:
Urgent - IOS crash when enableHybridAppEvents is enabled #745 (#746)

peterpeterparker committed:
release: v4.2.1

guylando committed:
Fix openUrl handling bugs and add support for cordova-ios-5 (#751)

* Update FacebookConnectPlugin.m

Fix https://github.com/jeduan/cordova-plugin-facebook4/issues/750

* Update README.md

* Upgraded facebook sdk to 5.0.2

* Fix calls to deprecated facebook sdk functions on android which caused runtime crash when called.

* Removing Bolts framework ios dependency which is not used in ios sdk v5 https://github.com/facebook/facebook-objc-sdk/commit/0bdd8502583d5094fde07ad0d6cca64202cf53b2

* Modernized FBSDKLoginKit usage according to
https://github.com/facebook/facebook-objc-sdk/commit/6c4ac12f42916c7e104469002973747a2e5c4c83
https://github.com/facebook/facebook-objc-sdk/commit/5eae2d7b5c8bd41edeac7bcf4f5f01e4db85dda3:

1. Replaced FBSDKLoginManagerRequestTokenHandler by FBSDKLoginManagerLoginResultBlock
2. Replaced logInWithReadPermissions and logInWithPublishPermissions by logInWithPermissions

Modernized FBSDKShareKit usage according to https://github.com/facebook/facebook-objc-sdk/commit/9b682bb292c32ca28111290fefe5fb8ea97c4531:

1. Updated FBSDKShareOpenGraphAction init to actionWithType
2. Replaced FBSDKGraphRequestHandler by FBSDKGraphRequestBlock

* removed irrelevant files

peterpeterparker committed:
release: v5.0.0

peterpeterparker committed:
feat(#639): use Cocoapods and bump up Facebook SDK version (#792)

peterpeterparker committed:
release: v6.0.0 (Cocoapods for iOS dependencies)

francescotonini committed:
Update iOS and Android SDKs (#805)

* Bump iOS SDK version to 5.6.0

* Bump Android SDK version to 5.5.1

peterpeterparker committed:
release: v6.1.0

sunda-amami committed:
Update plugin.xml (#808)

update pods

peterpeterparker committed:
doc: update iOS FB SDK v5.7.0

peterpeterparker committed:
lib: update default FB SDK Android v5.5.2 (#809)

peterpeterparker committed:
doc: cocoapods known erros (#810)

peterpeterparker committed:
release: v6.2.0

peterpeterparker committed:
Merge remote-tracking branch 'origin/master'

tetkosimi committed:
Update to CocoaPods CDN (#848)

https://blog.cocoapods.org/CocoaPods-1.7.2/

peterpeterparker committed:
feat: upgrade Facebook SDK iOS and Android (#849)

peterpeterparker committed:
doc: add table of contents

peterpeterparker committed:
doc: update table of contents

peterpeterparker committed:
doc: add sample repo

peterpeterparker committed:
doc: add sample repo

peterpeterparker committed:
release: v6.3.0

blackwolf12333 committed:
Fix hasAccessToken check (#853)

When an access token is expired this plugin would not
fetch a new one. This is invalid behaviour and resulted
in failed login attempts because the token was expired,
but still passed to the javascript code as a valid token.

This patch adds a check if the token is expired as well.

peterpeterparker committed:
release: v6.4.0

peterpeterparker committed:
docs: plugin is looking for maintainer or active fork

jeduan committed:
Deprecation notice

jeduan committed:
Update README.md
